packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -391,12 +392,15 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     const id = this._id;
     const { selectedIndices = [] } = this.state;
 
+    const activeElement = selectedIndices[0] !== undefined ? `${id}-list${selectedIndices[0]}` : undefined;
+
     return (
       <div className={styles.listWrapper} onKeyDown={this._onZoneKeyDown} ref={this._host} tabIndex={0}>
         <FocusZone
           ref={this._focusZone}
           direction={FocusZoneDirection.vertical}
-          defaultActiveElement={selectedIndices[0] !== undefined ? `#${id}-list${selectedIndices[0]}` : undefined}
+          defaultActiveElement={'#' + activeElement}
+          aria-activedescendant={activeElement}
```

Comment:
This seems to be adding more errors in accessibility audits. 

![image](https://user-images.githubusercontent.com/26070760/41479072-cc002d46-707e-11e8-9f20-e6301b763410.png)